puzzles.yaml

```diff
@@ -0,0 +1,20 @@
+app-id: uk.org.greenend.chiark.sgtatham.puzzles 
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: mosaic
+modules:
+  - imagemagick.yaml
+  - name: main
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://www.chiark.greenend.org.uk/~sgtatham/puzzles/puzzles-20230313.adf2a09.tar.gz
+        sha256: 32944cff9cee74f9b3e44d31c9e92492dafaae7b6477acbe3bc20b0775e60508
+      - type: patch
+        path: desktop.patch
+finish-args:
```

Comment:
`finish-args` is better when its above `modules`

Reply:
Fixed